frontend/src/shared/modules/filters/components/filterTypes/MultiSelectAsyncFilter.vue

```diff
@@ -33,6 +33,12 @@
               <span class="el-checkbox__inner" />
             </span>
           </div>
+          <template v-if="config.id === 'organizations'">
+            <span class="border rounded-md p-1 mr-2">
+              <img v-if="option.logo" :src="option.logo" class="h-5 w-5 flex items-center justify-center" :alt="option.label" />
```

Comment:
Can you also add `min-w-[16px]`. Some logos have a lower width and they show bad formatter if they don't have this class
<img width="316" alt="Screenshot 2023-10-03 at 12 16 13" src="https://github.com/CrowdDotDev/crowd.dev/assets/20134207/b3dd6919-50e8-4f16-922b-fa3c373cb278">